cmd/detect.go

```diff
@@ -109,24 +134,73 @@ var detectCmd = &cobra.Command{
 				Rows:    sensorRows,
 			}
 
-			tables := []table.Table{fanTable, sensorTable}
+			printTables([]table.Table{fanTable, sensorTable})
+		}
+
+		nvControllers := nvidia.GetDevices()
+
+		if len(nvControllers) > 0 {
+			ui.Println("=========== nvidia: ===========\n")
+		}
+
+		for _, ctrl := range nvControllers {
+
+			if len(ctrl.Fans) <= 0 && ctrl.Sensor == nil {
+				continue
+			}
+
+			ui.Printfln("> %s", ctrl.Identifier)
+			// TODO: print a label for the device from device.GetName() (e.g. "NVIDIA GeForce RTX 3060 Ti")?
 
-			for idx, table := range tables {
-				if table.Rows == nil {
-					continue
+			var fanRows [][]string
+			for _, fan := range ctrl.Fans {
+				pwmText := "N/A"
+				//pwm, err := fan.GetPwm() // FIXME: use this once it works
+				pwm := 69
+				var err error = nil
+				if err == nil {
+					pwmText = strconv.Itoa(pwm)
 				}
-				var buf bytes.Buffer
-				tableErr := table.WriteTable(&buf, tableConfig)
-				if tableErr != nil {
-					ui.Fatal("Error printing table: %v", tableErr)
+
+				// TODO: RPM?
+
+				//isAuto, _ := fan.IsPwmAuto()
+				isAuto := true
+
+				row := []string{
+					"", strconv.Itoa(fan.Index), fan.Label, pwmText, fmt.Sprintf("%v", isAuto),
 				}
-				tableString := buf.String()
-				if idx < (len(tables) - 1) {
-					ui.Print(tableString)
-				} else {
-					ui.Println(tableString)
+				fanRows = append(fanRows, row)
+			}
+			// TODO: RPM column, if supported
+			var fanHeaders = []string{"Fans   ", "Index", "Label", "PWM", "Auto"}
+
+			fanTable := table.Table{
+				Headers: fanHeaders,
+				Rows:    fanRows,
+			}
+
+			var sensorRows [][]string
+			if ctrl.Sensor != nil {
```

Comment:
Is there a reason the "fans" are a list yet the "sensor" is not?
Can we use a list for both, even if the lib doesn't support this yet?
I would like to align the logic that deals with table entries in general.